src/AlgebraicGeometry/ToricVarieties/CohomologyClasses/constructors.jl

```diff
@@ -54,11 +54,17 @@ julia> cohomology_class(d)
 Cohomology class on a normal toric variety given by 6*x3
 ```
 """
-function cohomology_class(d::ToricDivisor)
+function cohomology_class(d::ToricDivisor; quick::Bool = false)
+  if quick == false
     indets = gens(cohomology_ring(toric_variety(d)))
     coeff_ring = coefficient_ring(toric_variety(d))
     poly = sum(coeff_ring(coefficients(d)[k]) * indets[k] for k in 1:length(indets))
     return CohomologyClass(toric_variety(d), poly)
```

Comment:
```suggestion
    return CohomologyClass(toric_variety(d), poly, false)
```
See above, this must be called in the following way, since I removed the default.